libnavigation-base/src/main/java/com/mapbox/navigation/base/utils/DecodeUtils.kt

```diff
@@ -122,6 +122,15 @@ object DecodeUtils {
         return stepsGeometryDecodeCache.getOrDecode(legStep.geometry(), precision)
     }
 
+    @JvmStatic
```

Comment:
@Zayankovsky as you added the caching, could you please confirm clearing is done properly here?